new_icons/script.rb

```diff
@@ -0,0 +1,85 @@
+require 'csv'
+require 'nokogiri'
+require 'json'
+
+template = <<-TSX
```

Comment:
What does the `<<-TSX` tag do?

Reply:
It's the Ruby syntax for a multiline string:

https://www.rubyguides.com/2018/11/ruby-heredoc/